src/js/lib/client/src/index.js

```diff
@@ -267,6 +273,30 @@ Electron.app.on('web-contents-created', (event,contents) => {
 
 
 
+// =======================
+// === Project Manager ===
+// =======================
+
+const child_process = require('child_process')
+const fss = require('fs')
+
+async function run_project_manager() {
+    let bin_path = args.local
+    if (bin_path === undefined) {
```

Comment:
I'd use here `if (bin_path)` which would handle `undefined`, `null`, and empty string, which is what we want I think.